scalahost/sbt/src/main/scala/scala/meta/scalahost/sbt/ScalahostSbtPlugin.scala

```diff
@@ -22,15 +40,21 @@ object ScalahostSbtPlugin extends AutoPlugin {
 
   lazy val scalahostAllSettings =
     scalahostBaseSettings ++
-      scalahostInjectCompilerPluginSettings ++
+      inTestAndCompile(scalahostConfigSettings) ++
+      scalahostInjectScalacOptions ++
```

Comment:
I would group the settings based on the place they are scoped. Following this criterion, these two can be grouped together.